src/Triangle.cpp

```diff
@@ -1,77 +0,0 @@
-
-//  Triangle.cpp
-//  Delauney Mesh Generation
-//
-//  Created by Ben Rider on 1/15/25.
-//
-
-#include "Triangle.hpp"
-#include <cmath>
-
-
-Circle Triangle::computeCircumcircle(){
-    int A_idx, B_idx=0; 
-    //In the case that one of the edges, of our triangle has a slope of 0, 
-    //its perpendicular bisector will have an infinite slope, making is
-    //incompatible with our current method of calculating the center of our circle
-    //This loop ensures we will never choose a side with a slope of 0.
-
-    for(int i=0; i<3; i++){
-        if(this->Edges.at(i).slope==0.0){
```

Comment:
little inconsistent using `at` vs subscript `[]` through out the code. i prefer subscript as it is typically faster as it can be optimized as a direct memory access vs a function call and required bounds checks.  You will likely get bounds gets with `[]` in debug anyway